### DISCUSSION TITLE
Drug concentration in the tissue

### DISCUSSION THREAD

**Discussion by luisa-osantos** [Open-Systems-Pharmacology/Forum#1849] [Questions & Problems]
Dear OSP community,

I have a question regarding a project I am working on. I’m simulating an IV administration of a class II drug in mice. I have observed data for both plasma and liver concentrations, with the liver samples collected fresh and homogenized for quantification.
While I was able to simulate the plasma concentration well (red line and pink dots), the liver profile is a bit off (green line is the liver tissue concentration and blue line is the liver intracellular concentration). The model could not accurately predict the liver Cmax.
![image](https://github.com/user-attachments/assets/10145fce-56dd-418b-9a12-78f733c7898e)

I know this drug exhibits lysosomal trapping, and I have accounted for this in the model, but I’m still encountering issues with the liver profile. I have also tried optimizing permeability and partition coefficient without any significant improvement.
So my question is what other parameters or processes may be relevant for the liver profile/simulation?
Any comments would be highly appreciated, and please let me know if anything is unclear.

Thanks in advance,
Luisa

**Comment by tobiasK2001** [comment]
Dear Lusia,

overall I would say you simulated liver concentration is not so far of, given that it is often difficult to hit measured data. However, the time course looks quite different. Your model assumed that it is instantly high in the liver following the iv plasma profile. But in you measured data it is peaking after 12 hours. I wonder if you can try to Mimik that behavior by reducing the organ permeability (bring less in the liver) and including an OATP1 uptake transport (to bring it slower in again). Maybe you can play around with that a bit.

Best, tobias

**Reply by luisa-osantos** [reply]
Hello Tobias,

I’m a bit cautious about reducing the organ permeability because the drug has high lipophilicity (around 5), and when I tried to optimize organ permeability in PK-Sim, the optimal parameter value was always at the maximum value. So, I’m not sure what the best mechanistic choice is, to reducing this parameter or maximize it. I also looked into liver uptake for this drug but couldn’t find any data, and I’m unsure whether I can include this uptake transporter without specific information on the drug.

Best,
Luisa

**Reply by tobiasK2001** [reply]
Dear Luisa,

Yes if with that high lipophilicity one would expect a high organ permeability. But has your compound a pKA and maybe charged at physiological pH?
While optimizing, did you try all distribution models? If yes, what was the best?

Best, Tobias

**Reply by luisa-osantos** [reply]
Dear Tobias,

Yes, the compound is charged at physiological pH. Based on your comment, I tried different cellular permeability calculations, and the charge-dependent Schmitt method was the best at capturing tissue accumulation. Plasma concentration in shown by the red line and pink dots, and the liver profile in green line (liver tissue concentration) and blue line (liver intracellular concentration).
![image](https://github.com/user-attachments/assets/5ccb1cf2-8986-4073-a66e-ee8c5d75297f)
However, when I moved to the PO model in mice, there is an underprediction using this calculation method.
![image](https://github.com/user-attachments/assets/89f98ad4-2ade-4522-b74d-a2fd94b2f93e)
I am still trying to figure it out what is happening, but thank you for your input.

Best, 
Luisa

**Comment by tobiasK2001** [comment]
Hi Luisa,
indeed the liver tissue conce looks better now. For po, it looks like you have to fit specific intestinal permeability for mouse. The Charge dependent Schmitt method also reduces calculated intestinal permeability be reducing the mucosa cellular permeability. That might not be realistic and the cellular permeability needs to be reduced in liver only. The alternative would be to keep PK standard for cellular permeability and fit the live cellular permeability only. Than the po data should not be affected so strong. Let me know if this helps

Best, Tobias

**Reply by luisa-osantos** [reply]
Dear Tobias,

Thank you again for your input. I’m not sure I fully understand what you mean by keeping PK Standard for cellular permeability and fitting only the liver cellular profile. Does this mean I should run separate simulations for plasma and liver concentration profiles, or is there a setting that allows me to apply different methods for plasma and liver in the same simulation?
Once again, thank you for your time and help.

Best,
Luisa
